src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartBuilderTabSection.tsx

```diff
@@ -126,17 +127,21 @@ const ChartBuilderTabSection = ({
     [onTableUpdate, query, releaseVersionId, dataBlock],
   );
 
+  const chartExportRef = useRef(null);
+
   return (
-    <ChartBuilder
-      releaseVersionId={releaseVersionId}
-      data={table.results}
-      meta={meta}
-      initialChart={dataBlock.charts[0]}
-      tableTitle={dataBlock.heading}
-      onChartSave={handleChartSave}
-      onChartDelete={handleChartDelete}
-      onTableQueryUpdate={handleTableQueryUpdate}
-    />
+    <RefContext.Provider value={chartExportRef}>
```

Comment:
We can use `<RefContextProvider>` here instead, and then we don't need to instantiate a useRef ourselves. please can you do the same for the tests too, anywhere that we're using RefContext.Provider